third_party/move/move-vm/types/src/values/function_values_impl.rs

```diff
@@ -160,6 +165,13 @@ impl<'d, 'c, 'l> serde::de::Visitor<'d> for ClosureVisitor<'c, 'l> {
             .ctx
             .required_function_extension()
             .map_err(A::Error::custom)?;
+        let format_version = read_required_value::<_, u16>(&mut seq)?;
```

Comment:
Do we want this to be aligned with how move enums are represented, for consistency? 
If so, this would need to be varint?